src/components/Stats/Stats.js

```diff
@@ -46,4 +54,4 @@ RawStats.propTypes = {
   templateProps: PropTypes.object.isRequired,
 };
 
-export default autoHideContainerHOC(headerFooterHOC(RawStats));
+export default autoHideContainerHOC(RawStats);
```

Comment:
You should remove the autohideContainer as well. We want to handle that in the panel.